dev_docs/RELEASES.md

```diff
@@ -55,11 +39,12 @@ For patch releases, only the version on the existing major and minor version bra
 
 * Test plan
 
-  Create a github issue for testing the release branch (follow the GitHub issue template for the test plan), It should contain:
-  * A link to all PRs in the APM Server repository that need to be tested manually. Use the `test-plan*` labels and the version labels
-    to create an overview over the PRs that need testing. For example, [test plan link for 8.3.0](https://github.com/elastic/apm-server/issues?q=label%3Atest-plan+is%3Aclosed+label%3Av8.3.0).
-  * Add other test cases that require manual testing, such as test scenarios on ESS, that are not covered by automated tests or
-    OS compatibility smoke tests for supporting new operating systems.
+  Create a github issue for testing the release branch ([use the GitHub issue `test plan` template](https://github.com/elastic/apm-server/issues/new?assignees=&labels=test-plan&projects=&template=test-plan.md)), It should contain:
```

Comment:
What about including a recent example?

Reply:
Are you referring to the link on line 41 or 43? This link brings you to the new issue view with the test-plan template selected (there is a separate PR for updating the test plan). On line 43 instead there is a link to a 8.3.0 test plan, which looks like what you would like to see updated?

Reply:
I'd like an new link to a recent test plan issue, in addition to the existing link to the issue view with filtered with test-plan label

Reply:
Updated in [`43a62fe` (#12983)](https://github.com/elastic/apm-server/pull/12983/commits/43a62fea15407820201f21ed1d0db2a7246c386d)